### ISSUE TITLE
Does not recognize "if and only if"

### ISSUE THREAD

**Issue by vi** [killercup/english-lint#5]
```
$ echo 'A number is even if and only if remainder after division of it by two is zero' | english-lint
adverbs: 'only' (1:24-28)
wordy: 'remainder' (1:32-41)
```

I don't think "only" should be linted against if it is a part of "if and only if" expression.

**Comment by killercup** [comment]
Oh, very interesting case!

Fixing this might be tricky. We'd either need to have look-ahead/look-behind for this, or have a second phase where we eliminate false positives.

For now, can I suggest you write "A number is even iff the remainder after division of it by two is zero"? :)

(Also weird that the "wordy" list contains "remainder". Surely wasn't written with math in mind!)
